README.md

```diff
@@ -124,6 +124,9 @@ The `phrases` prop is an object that contains all the English language phrases c
   }),
 ```
 
+**Theming**
+react-dates comes with a set of scss variables that can be overridden to add your own project-specific theming. Override any variables found in `css/variables.scss` with your own and then import `~react-dates/css/styles.scss`.
```

Comment:
nit: scss -> SCSS. Also, it would be nice to provide a full example here to help people understand how to do this.